$NetBSD: patch-mplsetup.cfg,v 1.1 2022/06/16 15:24:40 ryoon Exp $

Instruct to use system Freetype

From: https://matplotlib.org/3.3.3/users/installing.html
-----------
FreeType and Qhull

Matplotlib depends on FreeType (>= 2.3), a font rendering library, and
on Qhull (>= 2015.2), a library for computing triangulations. By
default (except on AIX) Matplotlib downloads and builds its own copy
of FreeType (this is necessary to run the test suite, because
different versions of FreeType rasterize characters differently), and
uses its own copy of Qhull.

To force Matplotlib to use a copy of FreeType or Qhull already
installed in your system, create a setup.cfg file with the following
contents:
-----------

--- /dev/null	2021-01-17 13:10:59.949185604 +0900
+++ mplsetup.cfg	2021-01-17 13:55:23.673846857 +0900
@@ -0,0 +1,3 @@
+[libs]
+system_freetype = true
+system_qhull = true
